tools/dbTools.py

```diff
@@ -149,7 +119,7 @@ def insert_full_article(self, article, id_provider):
             RETURNING id;
             """,
             (article['title'], article['uri'], article['pdf'],
-             article['hash'], article.get('authors', ''), article['year'],
+             article['hash'], article.get('authors', ''), article.get('year'),
```

Comment:
is there a way to refactor this and the line below?

Reply:
I'm going to try something, but I'm not sure

Reply:
Not yet, but I'll switch to SQLalchemy soon. Sounds right?

Reply:
Sounds perfect 👍